### ISSUE TITLE
How to set a Pixbuf to a TreeStore column

### ISSUE THREAD

**Issue by haijin-development** [romgrk/node-gtk#132]
Hi.

I am trying to set an image to a `Gtk.TreeStore` store but I can not figure it out.
Python and Ruby do some type inference under the hoods I can not find an example using a GObject.Value

I am trying something like

```javascript
const gtkType = Gtk.ImageType.PIXBUF
const value = new GObject.Value()
value.init(gtkType)
value.setObject(pixbuf)
treeStore.setValue(iter, columnIndex, value)
```

but with no luck.

Is there an example I can grab?

**Comment by romgrk** [comment]
Hey @haijin-development,

I've uploaded one just now, it's here: [examples/gtk-tree-store.js](https://github.com/romgrk/node-gtk/blob/master/examples/gtk-tree-store.js)

By the way, when you provide some code, please consider posting a link to the full file you're working with, it saves me from having to recreate the whole context you're working with ^^

Let me know if you have more questions.

Demo:
![Screenshot from 2019-08-29 17-15-19](https://user-images.githubusercontent.com/1423607/63977104-e221c780-ca80-11e9-841f-553b69b9a4e4.png)

**Comment by donnisnoni95** [comment]
Yesterday I try to figure it out but fail

**Comment by romgrk** [comment]
@donnisnoni95 when I need to figure something like this I first check the C documentation for how to do it, because usually the API is a 1:1 match, e.g. C's `gtk_widget_show(...)` will be called with JS's `widget.show()`.
Then if I still have trouble I check how Python's PyGObject or Gnome's JS (aka gjs) is doing it. For this particular example I found the solution with PyGObject's help.

**Comment by donnisnoni95** [comment]
ya... I try from [this example](https://en.wikibooks.org/wiki/GTK%2B_By_Example/Tree_View/Tree_Models) but fail... Ya, I know about that thing and do that, the `gtk_widget_show`... thing

**Comment by haijin-development** [comment]
Hi @romgrk.

Thanks!

I used your example code and it worked.

Sorry for not posting the example code, I did not because it was not working and it had a lot of additional context that I thought would add noise. I will create a simplified use case the next time.

If its worth, here's the final working implementation of the code you provided integrated with the project I am working on:

https://github.com/haijin-development/node-sirens/blob/master/src/gui/views/ListView.js

Thanks again.
